nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -388,6 +413,25 @@ You can use it as a starting point and customize it for your OS or scheduler.
         return 0;
     }
 
+     int nrf_modem_os_mutex_init(void **mutex)
+    {
+        /* The function shall allocate and initialize a mutex and return its address
+           through the `mutex` parameter. If an address of an already allocated mutex is provided as
+           an input, the allocation part is skipped and the mutex is only reinitialized. */
```

Comment:
This line is not immediately clear to me. Ditto for the same sentence at line #274.

Reply:
The point is that if we get the address of a semaphore that is already initialized, we should reuse it instead of initializing a new semaphore, though we need to reset its initial count.